about.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
- [ ] As suggested by the previous reviewer, kindly consider adding the video link to your pull request summary as it is required. you can edit the pull request summary by clicking the three dots and then add the link of your loom video presentation.
- For recording the video you can use tools like [Loom](https://www.loom.com/) that let you share a private link to the recording, and configure a shot that shows your computer screen and your face at the same time in a small picture.

- Add the video link to your pull request description.